includes/code-analysis/excluded-symbol-names.md

```diff
@@ -0,0 +1,22 @@
+### Excluded symbol names
+
+You can exclude specific symbols, such as types and methods, from analysis. For example, to specify that the rule should not run on any code within types named `MyType`, add the following key-value pair to an *.editorconfig* file in your project:
+
+```ini
+dotnet_code_quality.CAXXXX.excluded_symbol_names = MyType
```

Comment:
It'd be awesome if we could have the `CAXXXX` replaced by the right ID but I don't know if the include can be parameterized.

Reply:
No, I don't think that's possible.